### PR TITLE
Deploy javadocs and sources

### PR DESCRIPTION
**Pull request by tkrille**
This is a requirement for the Central and really nice to have
for snapshots too.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -162,6 +162,9 @@
                 <configuration>
                     <file>${build.directory}/${project.build.finalName}.jar</file>
```
Comment (dacrome) [inline pom.xml]: Please change to `project.build.directory`

Comment (tkrille) [inline pom.xml]: Good catch, thanks!

**Comment by tkrille** [comment]
Updated

**Comment by dacrome** [comment]
Thank you!

**Pull request merged**